File: tags/2012.5/app/models/story_value.rb
class StoryValue < ActiveRecord::Base
  belongs_to :story
  belongs_to :story_attribute
  attr_accessible :value, :story_id, :story_attribute_id
  acts_as_audited

  validates_presence_of     :value
  validates_length_of       :value, :maximum => 4096, :allow_nil => true # Allow nil to workaround bug
end

File: src/app/story-value.ts
export class StoryValue {
  public id: number;
  public story_id: number;
  public story_attribute_id: number;
  public value: string;

  constructor(values: any) {
    this.id = values.id;
    this.story_id = values.story_id;
    this.story_attribute_id = values.story_attribute_id;
    this.value = values.value;
  }
}
